core/controller/src/main/scala/whisk/core/loadBalancer/LoadBalancerService.scala

```diff
@@ -184,20 +173,35 @@ class LoadBalancerService(config: WhiskConfig)(implicit val actorSystem: ActorSy
      * INVOKER MANAGEMENT
      */
 
-    private val pingConsumer = new KafkaConsumerConnector(config.kafkaHost, "health", "health")
-    pingConsumer.onMessage((topic, _, _, bytes) => {
-        val raw = new String(bytes, "utf-8")
-        PingMessage.parse(raw) match {
-            case Success(p: PingMessage) => invokerPool ! p
-            case Failure(t)              => logging.error(this, s"failed processing message: $raw with $t")
+    /** Create an test action on startup if it is not there. */
+    def createTestActionForInvokerHealth() = {
+        implicit val tid = TransactionId.loadbalancer
+        InvokerPool.testAction match {
+            case Some(action) =>
+                WhiskAction.get(db, action.docid).recover {
+                    case _: NoDocumentException => WhiskAction.put(db, action)
+                }.andThen {
+                    case Success(_) => logging.info(this, "Testaction for invokerHealth exists now.")
+                    case Failure(e) => logging.error(this, s"Error on creating testaction for invokerHealth: $e")
+                }
+            case None => logging.error(this, "Error on creating testaction for invokerHealth: Probably there were problems with the manifest.")
         }
-    })
+    }
+    createTestActionForInvokerHealth()
```

Comment:
Do we need to return a `Future` here or `Await` it? Otherwise this will race with the InvokerPool?